src/editors/array.js

```diff
@@ -393,8 +395,8 @@ JSONEditor.defaults.editors.array = JSONEditor.AbstractEditor.extend({
       else if(this.value.length === 1) {      
         this.remove_all_rows_button.style.display = 'none';  
 
-        // If there are minItems items in the array, hide the delete button beneath the rows
-        if(minItems || this.hide_delete_buttons) {
+        // If there are minItems items in the array, or configured to hide the delete_last_row button, hide the delete button beneath the rows
+        if(minItems || this.hide_delete_buttons || this.hide_delete_last_row_buttons) {
```

Comment:
this.hide_delete_buttons hides all of the delete buttons, so if that flag is set, the programmer doesn't need to set this.hide_delete_last_row_buttons.

If the programmer hast set neither this.hide_delete_buttons nor this.hide_delete_all_rows_buttons, and has set this.hide_delete_last_row_buttons, then it will hide the delete-last-row button, but will display the delete-all-rows button